_guestbook/alexisRemmers.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: Alexis
 image: https://lh3.googleusercontent.com/-Qb1HOXFDr4o/AAAAAAAAAAI/AAAAAAAAAEM/tXppNX3uQqU/s640/photo.jpg
-emoji: 🐼
+emoji: 🐹
```

Comment:
Sorry just no.

Reply:
agreed

Reply:
Pls, get this fixed right away, you're ruining the fun...